### PR TITLE
Re-enable CDS cache now that envoyfilter key is fixed

### PR DESCRIPTION
**Pull request by howardjohn**
Since https://github.com/istio/istio/pull/34404

### CODE REVIEW

File: pilot/pkg/features/pilot.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ var (
 
 	PushThrottle = env.RegisterIntVar(
 		"PILOT_PUSH_THROTTLE",
-		100,
+		1,
```
Comment (hzxuzhonghu) [inline pilot/pkg/features/pilot.go:63]: I guess this is for debug

Comment (howardjohn) [reply pilot/pkg/features/pilot.go:63]: oops, not intended to merge. Thanks!

**Review by hzxuzhonghu — APPROVED** [review]
Suggest not backport to 1.11, in case potential bug.

**Comment by howardjohn** [comment]
agreed, this will give 3mo to test CDs and rds

**Pull request merged**